tests/metrics/test_utilities.py

```diff
@@ -1,8 +1,40 @@
 from itertools import  product
 
+import numpy as np
+import pandas as pd
 import pytest
 
-from vivarium_public_health.metrics.utilities import QueryString
+from vivarium_public_health.metrics.utilities import (QueryString, to_years,
+                                                      get_susceptible_person_time, get_disease_event_counts)
+
+
+@pytest.fixture(params=((0, 100, 5, 1000), (20, 100, 5, 1000)))
+def ages_and_bins(request):
+    age_min = request.param[0]
+    age_max = request.param[1]
+    age_groups = request.param[2]
+    num_ages = request.param[3]
+
+    ages = np.linspace(age_min, age_max - age_groups/num_ages, num_ages)
```

Comment:
ooh I like this. So much better than manually making up data (my bad goto!)

Reply:
Building data is always hard.